packages/spruce-skill-server/config/default.ts

```diff
@@ -0,0 +1,288 @@
+		 * This can not be changed after you register your skill. Changing this value
+		 * will NOT update your slug
+		 */
+		SLUG: process.env.SLUG,
+		/**
+		 * 🌲🤖 Your Skill's description
+		 */
+		DESCRIPTION: process.env.DESCRIPTION,
+		/**
+		 * 🌲🤖 The port to start up
+		 */
+		PORT: process.env.PORT,
+		/**
+		 * 🌲🤖 The base url to your Skill's BE
+		 */
+		SERVER_HOST: process.env.SERVER_HOST,
+		/**
+		 * 🌲🤖 The vimeo video ID to show on your marketing page
+		 */
+		VIMEO_ID: process.env.VIMEO_ID,
+		/**
+		 * 🌲🤖 The base url to your Skill's FE
+		 */
+		INTERFACE_HOST: process.env.INTERFACE_HOST,
+		/**
+		 * 🌲🤖 If "true", ignores SSL errors when loading the FE
+		 */
+		INTERFACE_SSL_ALLOW_SELF_SIGNED:
+			process.env.INTERFACE_SSL_ALLOW_SELF_SIGNED === 'true',
+		/**
+		 * 🌲🤖 If "true", ignores SSL errors when loading the BE
+		 */
+		API_SSL_ALLOW_SELF_SIGNED: process.env.API_SSL_ALLOW_SELF_SIGNED === 'true',
+		WHITELABEL: process.env.WHITELABEL,
+		/**
+		 * 🌲🤖 If "true", enables crons
+		 * See controllers/cron
+		 */
+		RUN_CRONS: process.env.RUN_CRONS === 'true',
+		/**
+		 * 🌲🤖 If "true", enables debug routes
+		 * This should only be enabled in local and development environments
+		 */
+		ENABLE_DEBUG_ROUTES: process.env.ENABLE_DEBUG_ROUTES === 'true',
+		/**
+		 * 🌲🤖 The maximum depth to allow for GQL queries. Default 10
+		 * See https://github.com/stems/graphql-depth-limit for more info
+		 */
+		GRAPHQL_MAX_DEPTH: process.env.GRAPHQL_MAX_DEPTH
+			? +process.env.GRAPHQL_MAX_DEPTH
+			: 10,
+		/**
+		 * 🌲🤖 The maximum complexity to allow for GQL queries. Default 1500
+		 */
+		GRAPHQL_MAX_COMPLEXITY: process.env.GRAPHQL_MAX_COMPLEXITY
+			? +process.env.GRAPHQL_MAX_COMPLEXITY
+			: 1500,
+		/**
+		 * 🌲🤖 Enable Skills database
+		 * More info: https://developer.spruce.ai/#/orm
+		 */
+		DB_ENABLED: process.env.DB_ENABLED === 'true',
+		/**
+		 * 🌲🤖 Enable GraphQL endpoints
+		 */
+		GRAPHQL_ENABLED: process.env.GRAPHQL_ENABLED !== 'false',
+		/**
+		 * 🌲🤖 Enable GraphQL debugging and docs
+		 */
+		GRAPHIQL_ENABLED: process.env.GRAPHIQL_ENABLED === 'true',
+		/**
+		 * 🌲🤖 This variable can be checked to determine if we're running
+		 * in a test environment.
+		 */
+		TESTING: process.env.TESTING === 'true',
+		/**
+		 * 🌲🤖 The S3 bucket name for assets
+		 */
+		S3_BUCKET: process.env.S3_BUCKET || '',
+		/**
+		 * 🌲🤖 Your skills package.json name
+		 */
+		PACKAGE_NAME: packageJSON.name,
+		/**
+		 * 🌲🤖 Your skills package.json version
+		 */
+		PACKAGE_VERSION: packageJSON.version,
+		gqlOptions: {
+			gqlDir: `${baseDirectory}/gql`
+		},
+		sequelizeOptions: {
+			enabled: process.env.DB_ENABLED === 'true',
+			runMigrations: process.env.DB_MIGRATIONS === 'true',
+			modelsDir: `${baseDirectory}/models`,
+			migrationsDir: `${baseDirectory}/migrations`,
+			// Additional sequelize options
+			options: {
+				logging: process.env.ORM_LOGGING === 'true' ? console.log : false
+			}
+		},
+		bodyParserOptions: {
```

Comment:
can we use `typeof Koa.BodyParser` or something here?

Reply:
Will https://www.npmjs.com/package/@types/koa-bodyparser work?

Reply:
Yeah, the issue w/ setting an interface on an object literal is that you lose the exact type...so in this case autocomplete would just show me the interface type when I start typing `config.bodyParserOptions.` instead of showing me exactly what's defined.

Same thing w/ errors and settings...you'll see the TODO comment in those files.  In the future once TS catches up and supports exact types we should be able to do that